DigitalLearningSolutions.Web/Scripts/searchSortFilterAndPaginate/searchSortFilterAndPaginate.ts

```diff
@@ -161,15 +161,28 @@ export class SearchSortFilterAndPaginate {
 
   static updateResultCount(count: number): void {
     const resultCount = <HTMLSpanElement>document.getElementById('results-count');
-    const oldResultCountMessage = resultCount.innerHTML;
     resultCount.hidden = false;
     resultCount.setAttribute('aria-hidden', 'false');
     let newResultCountMessage = count === 1 ? '1 matching result' : `${count.toString()} matching results`;
+    newResultCountMessage = this.modifyResultCountMessageForScreenReaderIfUnchanged(
+      newResultCountMessage,
+      resultCount,
+    );
+
+    resultCount.innerHTML = newResultCountMessage;

```

Comment:
Does the html not get modified if you do this in the new function?

Reply:
Hmm, you can either get `resultCount` from the dom again within the new method (and extract a `getResultCount()` function), or you can rename the method to `GetNewResultCountMessage` or something. I think I'm okay with either, but in the latter case it would be worth adding a comment to explain what you're doing with the nbsp.